.github/workflows/tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13-dev"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
have you checked compatibility with the new 3.13?

Reply:
The latest pylint-django is not compatible with python 3.13, but the CI's tests are not catching that (2 tests are failing in the pylint-django pipeline, which doesn't mean that prospector is incompatible, there's just an issue in pylint-django).

Reply:
We already had python 3.13 tests passing (running on 3.13rc2 as far as I know)

Reply:
gotcha, thanks! :beer: